### DISCUSSION TITLE
alert details message add Preview Tools

### DISCUSSION THREAD

**Discussion by Ive4** [grafana/grafana#46267] [Alerting]
When I write summary or description, if I use variables I can't know the result of the current variables, if there is a preview function I will be able to get the result of the summary or description format before I receive the alert message.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
